v1/plugin/grpc_tls_test.go

```diff
@@ -175,15 +127,8 @@ func newGrpcOptsBuilder() *grpcOptsBuilder {
 	return &grpcOptsBuilder{}
 }
 
-func TestMain(m *testing.M) {
-	setUpTestMain()
-	retCode := m.Run()
-	tearDownTestMain()
-	os.Exit(retCode)
-}
-
 func TestIncorrectPluginArgsFail(t *testing.T) {
-	FocusConvey("Intending to start secure plugin server", t, func() {
+	Convey("Intending to start secure plugin server", t, func() {
 		setUpSecureTestcase(true, true)
 		Convey("omitting Cert Path from arguments will make plugin fail", func() {
```

Comment:
Not here - this relates to server's TLS certificate. System cert pool provides set of trusted certificates which are use to validate **client's** certificate.